src/base/solvers/solveCobraQP.m

```diff
@@ -573,7 +573,7 @@
             osense = 1;
        end
 
-        QPproblem.Q = 0.5*sparse(QPproblem.F);
+        QPproblem.Q = osense*0.5*sparse(QPproblem.F);
```

Comment:
I'm not sure why solveCobraQP should change the objective. 
The respective code has to ensure, that the F matrix is either positive semidefinite or negative semidefinite depending on the objective. 
This change would break a lot of code that considers this and formulates its F matrix accordingly. 
In addition, changing this solely on gurobi would make gurobi return results different to any other solver, but as I said, I would not change this at all, as it is changing the objective. I.e. it is impossible to maximise the sum of quadratic fluxes. But with this change, it would actually minimise them (valid), which is not what was intended.